source/extensions/common/wasm/wasm.cc

```diff
@@ -2180,6 +2185,7 @@ void Wasm::getFunctions() {
 #undef _GET
 
 #define _GET_PROXY(_fn) wasm_vm_->getFunction("_proxy_" #_fn, &_fn##_);
+  _GET_PROXY(onValidateConfiguration);
```

Comment:
Since this is expected to be used in control planes, I'm not sure if there is any reason to plug this all the way inside Envoy (i.e. it should probably be limited to Proxy-WASM SDK, but not host implementation).

Reply:
It doesn't hurt to have it here, so perhaps we can leave it until we figure out how this is going to be used.  @mandarjog